applications/serial_lte_modem/doc/slm_description.rst

```diff
@@ -539,12 +540,11 @@ If you have an nRF52 Series DK running a client application, you can also use th
 |test_sample|
 
 1. |connect_kit|
-#. :ref:`Connect to the kit with LTE Link Monitor <lte_connect>`.
-   If you want to use a different terminal emulator, see ref:`slm_connecting_91dk_pc`.
+#. :ref:`Connect to the kit with nRF Connect Serial Terminal <serial_terminal_connect>`.
```

Comment:
I wonder if this should indicate that you should use a specific line ending? If I remember correctly, this did not work out-of-the-box.

Reply:
CRLF is the default line ending. My memories were incorrect in this case.